### DISCUSSION TITLE
Можно ли включить автостарт программы при загрузке устройства?

### DISCUSSION THREAD

**Discussion by SubZero77** [dovecoteescapee/ByeDPIAndroid#157] [Q&A]
Можно ли сделать автозапуск при загрузке устройства?

**Comment by zimbabwe303** [comment]
В ByeByeDpi от @romanvht есть такое: https://github.com/romanvht/ByeDPIAndroid.

@dovecoteescapee куда-то делся, [пулл-реквест](https://github.com/dovecoteescapee/ByeDPIAndroid/pull/136) от @romanvht с этими функциями висит ещё с 3 сентября.
